lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
Glance through https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/string.ex to see what I mean.